### PR TITLE
anormDB: move the DB schema out of the code in the case of MySQL

### PR DESCRIPTION
**Pull request by jfeltesse-mdsol**
The user will have to load the schema manually, just like when using Cassandra.

This is following a discussion on the chat about how docker-zipkin has to fetch the cassandra schema at build time while a mysql based version wouldn't have such a step.
The benefit of manually loading the schema is that we wouldn't need the collector service to start before the query service so that it can setup the schema auto-magically.

Note that the code changes are untested, I am not a scala hacker.
In other words,

enhanced-14836-1414320930-8

@adriancole have a look when you have a minute please.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
PS I'm offline until Monday. Anyone else can feel free to look at this,
else I will then.

**Comment by codefromthecrypt** [comment]
@jfeltesse-mdsol so I think we still want this to install the schema, even if it is reading from files. (preserving the install=true contract). I'll add work on your commit to do that.

**Comment by codefromthecrypt** [comment]
ps I think these should be the same file. the only reason the indexes weren't in the default formerly was that index syntax isn't portable.

**Comment by codefromthecrypt** [comment]
actually I see what you mean. mysql needs a separate step to install the database anyway, and probably prod usage wouldn't want to give it access to change its schema.

**Comment by codefromthecrypt** [comment]
#705

**Comment by codefromthecrypt** [comment]
Thanks for the start! I added polish on the other.
